src/hooks/use-sonification-scheduler.ts

```diff
@@ -0,0 +1,101 @@
+// hooks/useSonificationScheduler.ts
+import { useCallback } from "react";
+import * as Tone from "tone";
+import { interpolateBins, mapPitchFractionToFrequency, mapValueToStereoPan } from "../components/graph-sonification-utils";
+import { isUnivariateDotPlot } from "../utils/utils";
+
+type Props = {
+  selectedGraph: any;
+  binValues: any;
+  pitchFractions: number[];
+  timeFractions: number[];
+  timeValues: number[];
+  primaryBounds: any;
```

Comment:
Would it be relatively easy to replace these instances of `any` with more specific types? I think `selectedGraph` could at least be `ICODAPGraphModel`, right?